.gitignore

```diff
@@ -65,3 +65,4 @@ util/Win_Check_Output.txt
 secrets.tar
 id_rsa_*
 /.vs
+lib/chibios-contrib
```

Comment:
Were you having trouble with this? Our `make git-submodule` should be able to fix it. We need to be able to track changes to that folder :)

Reply:
submodules are synced but there is nothing new to git add so i cannot recommit :S

Reply:
Cool! Can you remove this line then? Just meant tracking changes in general, not with this specific PR :)

Reply:
Sorry this is embarrassing. I just realized what the issue was:)

Reply:
I think this should be ok now:)